src/SnoopCompile.jl

```diff
@@ -71,6 +71,51 @@ if VERSION >= v"1.2.0-DEV.573"
 
 end
 
+"""
+    timesum(snoop)
+
+Calculates and prints the total time measured by a snoop macro
+
+# Examples
+To benchmark the loadTime and RunTime of a package:
```

Comment:
I worry that this is misleading. Much of the load time is spent inserting methods into method tables (https://discourse.julialang.org/t/how-to-debug-compilation-time-and-loading-time-of-packages/31583/4?u=tim.holy); for run time, keep in mind that `@snoopi` only measures inference time, and the run time of the tests might depend on computation time as well as inference time.

It would be accurate to say that these measure the *inference time* from loading the package and running the tests.

Reply:
Here I meant inferring, but I need to do clarification. 

So for general compiling time benchmarking, do you recommend `snoopi` or `snoopc`?

Reply:
They measure very different things. The reason `@snoopi` is more aligned with the purpose of this package is that currently Julia can only cache inference results; hence inference time, rather than compile time, is the benchmark that you can expect to see reduced by adding precompile statements.